shakenfist/virt.py

```diff
@@ -793,3 +862,53 @@ def get_console_data(self, length):
             'Client requested %d bytes of console log, returning %d bytes'
             % (length, len(d)))
         return d
+
+
+class Instances(object):
```

Comment:
This is not a concept that I find easy to like. If we are going OO maybe this should be a factory class of some sort?